conf/mini_provisioner/v2/ha.config.tmpl.3-node

```diff
@@ -21,6 +21,13 @@ server_node:
         netmask: TMPL_NETMASK
     s3_instances: '1'
     type: TMPL_TYPE
+    stonith:
```

Comment:
Are these keys present in the mini-provisioning confstore already?

Reply:
Done changed.